man/bgCol.default.Rd

```diff
@@ -4,14 +4,12 @@
 \alias{bgCol.default}
 \title{create a background color}
 \usage{
-bgCol.default(col, ...)
```

Comment:
took this out because `bgCol` should only have 1 arg, right? unless we are supporting shading. If we are (like w/ `density`) it should be an explicit arg, not `...`